docs/content/en/docs/getting-started/_index.md

```diff
@@ -15,58 +25,266 @@ In your local environment the following tools are required for provisioning a K8
+
+Once all the pods are in the `Running` or `Completed` state, you can check the health of your K8ssandra cluster.
+
+{{% alert title="Tip" color="success" %}}
+The pod `k8ssandra-reaper-k8ssandra-schema-xxxxx` runs once and does not persist.
+{{% /alert %}}
+
+The actual Cassandra node name from the listing above is `k8ssandra-dc1-default-sts-0`. If you've configured multiple nodes, `sts-0` will increment.
+
+We'll use node name `k8ssandra-dc1-default-sts-0` throughout the following sections.
+
+To check the health of your K8ssandra cluster:
+
+1. Confirm that the Cassandra operator is `Ready`:
+
+    ```bash
+    kubectl describe CassandraDataCenter dc1 | grep "Cassandra Operator Progress:"
+       Cassandra Operator Progress:  Ready
+    ```
+
+1. Get K8ssandra superuser credentials:
+
+    * K8ssandra superuser name:
+
+        ```bash
+        kubectl get secret k8ssandra-superuser -o jsonpath="{.data.username}" | base64 --decode | more
+        k8ssandra-superuser
+        (END)
+        ```
+
+    * K8ssandra superuser password:
+
+        ```bash
+        kubectl get secret k8ssandra-superuser -o jsonpath="{.data.password}" | base64 --decode | more
+        PGo8kROUgAJOa8vhjQrE49Lgruw7s32HCPyVvcfVmmACW8oUhfoO9A
+        (END)
+        ```
+
+1. Run `nodetool status`, using the Cassandra node name `k8ssandra-dc1-default-sts-0`, and passing the superuser name and password. Verify that the node is in the state `UN` or Up Normal:
+
+    ```bash
+    kubectl exec -it k8ssandra-dc1-default-sts-0 -c cassandra -- nodetool -u k8ssandra-superuser -pw PGo8kROUgAJOa8vhjQrE49Lgruw7s32HCPyVvcfVmmACW8oUhfoO9A status
+    Datacenter: dc1
+    ===============
+    Status=Up/Down
+    |/ State=Normal/Leaving/Joining/Moving
+    --  Address      Load       Owns    Host ID                               Token                                    Rack
+    UN  10.244.1.12  215.3 KiB  ?       75e52e51-edc9-49f8-84f6-f044999ac130  -1080085985719557225                     default
+
+    Note: Non-system keyspaces don't have the same replication settings, effective ownership information is meaningless
+    ```
+
+{{% alert title="Tip" color="success" %}}
+Save the superuser name and password for use in future steps.
+{{% /alert %}}
+
+## Stopping and starting Cassandra {#cassandra-operations}
+
+Before shutting down your Kubernetes cluster, you'll want to make sure you cleanly shut down your Cassandra datacenters. You can do that using the `kubectl patch` command and setting the `spec:stopped` property to either `true` (stopped) or `false` (running).
+
+### Shut down Cassandra
+
+To shut down a Cassandra datacenter:
+
+```bash
+kubectl patch cassdc <datacenter-name> --type merge -p '{"spec":{"stopped":true}}'
+```
+
+Example:
+
+```bash
+kubectl patch cassdc dc1 --type merge -p '{"spec":{"stopped":true}}'
+cassandradatacenter.cassandra.datastax.com/dc1 patched
+```
+
+### Start up Cassandra
+
+To start up a Cassandra datacenter
```

Comment:
Starting a stopped Cassandra datacenter is the reverse of stopping one. Setting the `stopped` parameter to `false` will cause all pods to be restarted with their data intact.

```bash
$ kubectl get cassdc
NAME   AGE
dc1    18h

$ kubectl patch cassdc dc1 --type merge -p '{"spec":{"stopped":false}}'
```